components/FlashCard.js

```diff
@@ -29,6 +59,17 @@ export default function FlashCard({ card, onMarkCorrect, collections }) {
     onMarkCorrect(card.id);
   }
 
+  function handleToggleButton(event) {
+    event.stopPropagation();
+    setIsClicked((prev) => !prev);
+  }
```

Comment:
See above - I suggest to be a bit more specific with your naming:

```suggestion
  function handleToggleDeletePrompt(event) {
    event.stopPropagation();
    setIsDeletePromptVisible((prev) => !prev);
  }
```